src/BlockInfo.cpp

```diff
@@ -733,7 +726,6 @@ cBlockInfo::cBlockInfoArray::cBlockInfoArray()
 	Info[E_BLOCK_POWERED_RAIL                 ].m_Transparent = true;
 	Info[E_BLOCK_PUMPKIN_STEM                 ].m_Transparent = true;
 	Info[E_BLOCK_PURPLE_SHULKER_BOX           ].m_Transparent = true;
-	Info[E_BLOCK_PURPUR_DOUBLE_SLAB           ].m_Transparent = true;
```

Comment:
So we just need the double slab changes from here and we'll fix the door issue right?

Reply:
Should be.